packages/gatsby-plugin-gatsby-cloud/src/__tests__/gatsby-browser.js

```diff
@@ -103,11 +103,11 @@ describe(`Preview status indicator`, () => {
 
     await waitFor(() => {
       if (action) {
-        // Initial poll fetch, initial load trackEvent, and trackEvent after action
-        expect(window.fetch).toBeCalledTimes(3)
+        // intial page data fetch, initial poll fetch, initial load trackEvent, and trackEvent after action
+        expect(window.fetch).toBeCalledTimes(5)
       } else {
         // Initial poll fetch for build data and then trackEvent fetch call
-        expect(window.fetch).toBeCalledTimes(2)
+        expect(window.fetch).toBeCalledTimes(3)
```

Comment:
I think it's still missing one call in each. The first describes 4 calls and the second describes 2 calls, rather than 5 and 3.